### PR TITLE
Cockpit 191

### PR DESCRIPTION
**Pull request by mvollmer**
Rendered: https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-191.html

### CODE REVIEW

File: _posts/2019-04-03-cockpit-191.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Cockpit can now help you with enabling certain security vulnerability mitigations.
+
+### Try it out
+
+Cockpit 191 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/191)
+ * [Fedora 29](https://bodhi.fedoraproject.org/updates/cockpit-191-1.fc29)
```
Comment (martinpitt) [inline _posts/2019-04-03-cockpit-191.md]: It's available for Fedora 30 as well.

File: _posts/2019-04-03-cockpit-191.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+summary: Cockpit with Machines and Networking improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 191.
+
+### Machines: iSCSI Storage pools
+
+You can now add a iSCSI target as a storage pool for your virtual
```
Comment (martinpitt) [inline _posts/2019-04-03-cockpit-191.md]: a"n" iSCSI target

File: _posts/2019-04-03-cockpit-191.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Notifications about virtual machines have been streamlined and use a
+common presentation now.
+
+![Notifications](/images/machines-notifications.png)
+
+### Network: Ports in the Firewall
+
+In addition to services, which represent a predefined set of ports,
+you now can now open individual ports in the firewall by directly
+specifying their number.
```
Comment (martinpitt) [inline _posts/2019-04-03-cockpit-191.md]: ... their number, name, or a range.

File: _posts/2019-04-03-cockpit-191.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+![Firewall ports](/images/networking-ports.png)
+
+### System: CPU security mitigation
+
+Cockpit can now help you with enabling certain security vulnerability mitigations.
```
Comment (martinpitt) [inline _posts/2019-04-03-cockpit-191.md]: perhaps: "..., starting with the [nosmt option](https://access.redhat.com/security/vulnerabilities/L1TF)." ?

File: _posts/2019-04-03-cockpit-191.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2019-04-03-cockpit-191.md]: Fixed

File: _posts/2019-04-03-cockpit-191.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2019-04-03-cockpit-191.md]: Fixed

File: _posts/2019-04-03-cockpit-191.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2019-04-03-cockpit-191.md]: Hmm, I omitted all the possibilities to keep it simply. I think just "number" gets the point across just fine.

File: _posts/2019-04-03-cockpit-191.md (hunk 4 above)
Comment (mvollmer) [reply _posts/2019-04-03-cockpit-191.md]: Yes, good idea. Fixed.

**Comment by mvollmer** [comment]
Rendered: https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-191.html

**Comment by garrett** [comment]
The notifications screenshot is incorrect.

**Comment by mvollmer** [comment]
> The notifications screenshot is incorrect.

What about this gif:

machines-notification-stack

**Comment by garrett** [comment]
@mvollmer: The GIF is also out of date. I need to make another.

Also, the ports dialog is out of date. There were a couple of very minor fixes since. (It should have an oxford comma, for example.)

**Comment by garrett** [comment]
Preview of the new images @ https://garrett.github.io/cockpit-project.github.io/blog/cockpit-191.html — the biggie is the animation.

I tried to do one to show off the scrolling too, but it was far too huge in size. I managed to get this GIF down in filesize, but it's still one of the chunkiest @ 290 kb. I guess recompressing the others kinda made up for it... a little.

**Comment by garrett** [comment]
Note: I did not apply the text changes. Please select the ones you want and merge them in.

File: _posts/2019-04-03-cockpit-191.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+In addition to services, which represent a predefined set of ports,
+you now can now open individual ports in the firewall by directly
+specifying their number.
+
+![Firewall ports](/images/networking-ports.png)
+
+### System: CPU security mitigation
+
+Cockpit can now help you with enabling certain security vulnerability
+mitigations, starting with the [nosmt option](https://access.redhat.com/security/vulnerabilities/L1TF).
```
[the remark this answers is not in the document]
Comment (martinpitt) [reply _posts/2019-04-03-cockpit-191.md]: Let's not do that (let's talk in private once you are on IRC)

File: _posts/2019-04-03-cockpit-191.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+You can now add an iSCSI target as a storage pool for your virtual
+machines.
+
+![iSCSI storage pool](/images/machines-iscsi.png)
+
+### Machines: Better notifications
+
+Notifications about virtual machines have been streamlined and use a
+common presentation now.
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2019-04-03-cockpit-191.md]: We regularly use "now" in tghe release notes. We can stop that any time, but then we should remove all references in this note.

**Pull request merged**